_includes/projects/card-in-list.html

```diff
@@ -5,18 +5,23 @@
     <dt class="hidden-from-ui">Summary</dt>
     <dd class="main">{{ project.short_description }}</dd>
 
+    {% comment %}
```

Comment:
Good question! I thought I should have described that in the PR.

Project tags used to be displayed (like on the actual version of the website). I feel it's unnecessary on the homepage to see them as it adds more noise than value.

`hidden-from-ui` is a CSS class to hide some labels because their corresponding value should be explicit enough about what they are.

![image](https://cloud.githubusercontent.com/assets/138627/23943423/ba71a0c2-096f-11e7-973c-7bcbce35205e.png)